metrics_service/tools/performance_tests/RESULTS_HTTP.md

```
@@ -0,0 +1,269 @@
+
+| Metric | Value |
+|--------|-------|
+| Requests received | 231 |
+| Responses sent | 231 |
+| Mean latency (server-side) | 1004.8ms |
+| CPU time used | 29.300s |
+| RSS memory (end) | 83.9 MB |
+
+---
+
+## Medium Run Detail
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)